### PR TITLE
chore: update samples cleanup

### PR DESCRIPTION
**Pull request by AVaksman**

### CODE REVIEW

**Review by sofisl — APPROVED** [review]

**Review by skuruppu — APPROVED** [review]

File: samples/system-test/spanner.test.js [hunk 1]
```diff
@@ -69,7 +69,7 @@ const GAX_OPTIONS = {
 async function deleteStaleInstances() {
   const [instances] = await spanner.getInstances({
     filter:
-      '(labels.gcloud-sample-tests:true) OR (labels.cloud_spanner_samples:true)',
+      '(labels.node-sample-tests:true) OR (labels.cloud_spanner_samples:true)',
```
Comment (stephenplusplus) [inline samples/system-test/spanner.test.js:72]: Can we combine the variables with the string here to make sure they’re consistent?

**Review by stephenplusplus — CHANGES_REQUESTED** [review]

File: samples/system-test/spanner.test.js (hunk 1 above)
Comment (AVaksman) [reply samples/system-test/spanner.test.js]: done with `'node-sample-tests'`, the `'cloud_spanner_samples'` label is from the [sample](https://github.com/googleapis/nodejs-spanner/blob/e40abaffadce6bd92f5ecbac7d054d4a1ca1c928/samples/instance.js#L44) itself.

**Review by stephenplusplus — APPROVED** [review]

File: samples/system-test/spanner.test.js [hunk 2]
```diff
@@ -67,9 +67,9 @@ const GAX_OPTIONS = {
 };
 
 async function deleteStaleInstances() {
+  console.log(`(labels.${LABEL}:true) OR (labels.cloud_spanner_samples:true)`);
```
Comment (stephenplusplus) [inline samples/system-test/spanner.test.js:70]: Can we get rid of this?

**Review by stephenplusplus — CHANGES_REQUESTED** [review]

File: samples/system-test/spanner.test.js (hunk 2 above)
Comment (AVaksman) [reply samples/system-test/spanner.test.js]: Yes, of course.
Done!

**Review by stephenplusplus — APPROVED** [review]

**Pull request merged**